core/object/object.h

```diff
@@ -677,6 +677,8 @@ class Object {
 	Object(bool p_reference);
 
 protected:
+	StringName _translation_domain;
```

Comment:
We don't really make properties available like that, especially not in core. You can use the setter/getter from base class to manage the domain.

Reply:
This is only available to subclasses.

`_translation_domain` is the storage space for the settled translation domain. Subclasses can change how to use this space by overriding `get_translation_domain()` and `set_translation_domain()`.

- `Object` uses it as a plain old property.
- `Node` uses it as a cache.

`StreamPeer` uses protected members too.